src/api.py

```diff
@@ -0,0 +1,18 @@
+import requests 
+import hashlib
+def fetch(url):
```

Comment:
We like to format the code using PEP8 conventions. We also group imports by origin (standard lib, third-party, own imports)

We separate also top level things (functions, classes) with two empty lines.